android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -68,6 +78,88 @@
   Map<String, Object> constants;
   AsyncTask<Void, Void, Map<String, Object>> futureConstants;
 
+  private boolean isDebug = true;
+
+  private boolean isTelephony = true;
+
+  private boolean isCheckPackage = true;
+
+  private List<String> mListPackageName = new ArrayList<>();
+
+  private static final String IP = "10.0.2.15";
+
+  private static final int MIN_PROPERTIES_THRESHOLD = 0x5;
+
+  private static final String[] DEVICE_IDS = {
+          "000000000000000",
+          "e21833235b6eef10",
+          "012345678912345"
+  };
+
+
+  private static final String[] IMSI_IDS = {
+          "310260000000000"
+  };
+
+  private static final String[] GENY_FILES = {
+          "/dev/socket/genyd",
+          "/dev/socket/baseband_genyd"
+  };
+
+  private static final String[] ANDY_FILES = {
+          "fstab.andy",
+          "ueventd.andy.rc"
+  };
+
+  private static final String[] NOX_FILES = {
+          "fstab.nox",
+          "init.nox.rc",
+          "ueventd.nox.rc"
+  };
+
+  private static final String[] QEMU_DRIVERS = {"goldfish"};
+
+  private static final String[] PIPES = {
+          "/dev/socket/qemud",
+          "/dev/qemu_pipe"
+  };
+
+  private static final String[] PHONE_NUMBERS = {
```

Comment:
Since these are used for emu checking, maybe EMU_PHONE_NUMBERS

Reply:
renamed
https://github.com/react-native-community/react-native-device-info/pull/721/commits/72aa24ded9aa61ae5d380f3138b4928d9bbab7d0